spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/stream/SkipperStreamDeployerTests.java

```diff
@@ -106,19 +106,19 @@ public void testInstallUploadProperties() {
 	@Test
 	public void testFileResourceProcessing() throws MalformedURLException{
 		Resource resource = new UrlResource("file:/springcloudstream/file-source-kafka-10-1.2.0.RELEASE.jar");
-		assertThat(SkipperStreamDeployer.getResourceWithoutVersion(resource)).isEqualTo("file:/springcloudstream/file-source-kafka-10.jar");
+		assertThat(SkipperStreamDeployer.getResourceWithoutVersion(resource)).isEqualTo("file:/springcloudstream/file-source-kafka-10-1.2.0.RELEASE.jar");
```

Comment:
the method is getResourceWithoutVersion but yet it will return with the version for the 'generic' http resource.  Will look to change some names around.